### ISSUE TITLE
Keycard: Noticeable delay in starting flows

### ISSUE THREAD

**Issue by glitchminer** [status-im/status-desktop#13242]
# Bug Report

## Description

There is a noticeable delay in the detection of the card reader at the beginning of flows that was not in previous versions.

## Steps to reproduce

- Go to keycard settings
- Click on check what is on a keycard
- Connect reader with card when prompted, note delay
- Close popup
- Click on check what is on a keycard again
- Note delay in recognition with card

#### Actual behavior

https://github.com/status-im/status-desktop/assets/50769329/33eaf70d-2c79-46ea-bf3f-cc81baf01995

### Additional Information

- Status desktop version: Master (local)
- Operating System: Mac 14.2.1

**Comment by saledjenic** [comment]
@glitchminer I've checked this and it acts much faster on my machines (tested on Mac and Windows) than what I see in your video. 
Some delays might be noticeable when closing the popup after you provide the pin, in that case we're doing some additional checks to sync the keycard's data according to the app's data, but before that, it should be just opening and closing the popup. 
But even without providing a pin opening popup requires running a keycard flow, which makes me think that the communication between the app and the keycard on your end is a bit lower. There should not be another reason for that. 

Basically, we don't have what to do to improve that.

@anastasiyaig @lukaszso could you check on your end if you see that delay or not?

**Comment by alaibe** [comment]
@benjthayer could you help and provide a visual indicator that something is loading?

**Comment by benjthayer** [comment]
> @benjthayer could you help and provide a visual indicator that something is loading?

Added to my list, thanks! 👍
